packaging/windows/provision.sh

```diff
@@ -39,29 +39,17 @@ function download_wheels() {
 function download_resources() {
 
     GITHUB_USERNAME=$1
-    GITHUB_PASSWORD=$2
+    GITHUB_TOKEN=$2
 
     mkdir -p packaging/source/{python,types,scripts,plugins}
     pushd packaging/source/python
-        curl -L http://gigaspaces-repository-eu.s3.amazonaws.com/org/cloudify3/components/Python279_x32.tar.gz -o /tmp/Python279_x32.tar.gz
+        curl -L https://cloudify-release-eu.s3.amazonaws.com/cloudify/components/Python279_x32.tar.gz -o /tmp/Python279_x32.tar.gz
         tar -zxvf /tmp/Python279_x32.tar.gz --strip-components=1
     popd
 
     # Downloading types.yaml
     pushd packaging/source/types
-        curl -LO http://www.getcloudify.org/spec/cloudify/${CORE_TAG_NAME}/types.yaml
-    popd
-
-    # Downloading Scripts
-    pushd packaging/source/scripts
-        curl -LO https://raw.githubusercontent.com/cloudify-cosmo/cloudify-manager/${CORE_BRANCH}/resources/rest-service/cloudify/fs/mkfs.sh
-        curl -LO https://raw.githubusercontent.com/cloudify-cosmo/cloudify-manager/${CORE_BRANCH}/resources/rest-service/cloudify/fs/fdisk.sh
-        curl -LO https://raw.githubusercontent.com/cloudify-cosmo/cloudify-manager/${CORE_BRANCH}/resources/rest-service/cloudify/fs/mount.sh
-        curl -LO https://raw.githubusercontent.com/cloudify-cosmo/cloudify-manager/${CORE_BRANCH}/resources/rest-service/cloudify/fs/unmount.sh
-        curl -LO https://raw.githubusercontent.com/cloudify-cosmo/cloudify-manager/${CORE_BRANCH}/resources/rest-service/cloudify/policies/host_failure.clj
-        curl -LO https://raw.githubusercontent.com/cloudify-cosmo/cloudify-manager/${CORE_BRANCH}/resources/rest-service/cloudify/policies/threshold.clj
-        curl -LO https://raw.githubusercontent.com/cloudify-cosmo/cloudify-manager/${CORE_BRANCH}/resources/rest-service/cloudify/policies/ewma_stabilized.clj
-        curl -LO https://raw.githubusercontent.com/cloudify-cosmo/cloudify-manager/${CORE_BRANCH}/resources/rest-service/cloudify/triggers/execute_workflow.clj
+        curl -LO http://cloudify.co/spec/cloudify/${CORE_TAG_NAME}/types.yaml
```

Comment:
This means that `types.yaml` has to be pre-uploaded to S3.
You won't always use the newest file.

Reply:
Well in daily build of the cli yes, if building from a branch the 'CORE_TAG_NAME' will be set to the closed version branch by the bump process.
And I am not sure if the types yaml is actually needed for the cli